scripts/compress_images.py

```diff
@@ -0,0 +1,196 @@
+# coding: utf-8
+#
+# Copyright 2025 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to compress images in the repository using GraphicsMagick."""
+
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+import tempfile
+
+from PIL import Image
+from typing import List, TypedDict, Union
+
+ALL_IMAGES_EXTENSIONS = {'.png', '.jpg', '.jpeg', '.webp'}
```

Comment:
Have this be defined as a sum of the two sets below, so that you don't have a possibility of it differing from what's in the two sets below.